pkg/cmd/auth/login/login.go

```diff
@@ -168,24 +182,26 @@ func loginRun(opts *LoginOptions) error {
 				return err
 			}
 
-			username, err := api.CurrentLoginName(apiClient, hostname)
-			if err != nil {
-				return fmt.Errorf("error using api: %w", err)
-			}
-			var keepGoing bool
-			err = prompt.SurveyAskOne(&survey.Confirm{
-				Message: fmt.Sprintf(
-					"You're already logged into %s as %s. Do you want to re-authenticate?",
-					hostname,
-					username),
-				Default: false,
-			}, &keepGoing)
-			if err != nil {
-				return fmt.Errorf("could not prompt: %w", err)
-			}
+			if opts.Interactive {
```

Comment:
This avoids the “Do you want to re-authenticate?” prompt in interactive mode, but still executes `client.ValidateHostCfg()` using the old token in non-interactive mode and discards its result.

If the existing token is valid, could we exit with some kind of success message? Or, if we want to unconditionally start the login flow, we could skip `ValidateHostCfg()` in non-interactive mode.